app/src/main/java/org/fossasia/susi/ai/skills/skilllisting/contract/ISkillListingView.kt

```diff
@@ -8,6 +8,6 @@ import org.fossasia.susi.ai.rest.responses.susi.SkillData
  */
 interface ISkillListingView {
     fun visibilityProgressBar(boolean: Boolean)
-    fun updateAdapter(skills: ArrayList<Pair<String, List<SkillData>>>)
+    fun updateAdapter(metricsList: ArrayList<Pair<String, List<SkillData>?>> = ArrayList())
```

Comment:
Can `ArrayList<Pair<String, List<SkillData>?>>` be converted to object

Reply:
Using a data class?

Reply:
Yes

Reply:
is this converted @arundhati24 ?

Reply:
Yes @batbrain7 .

Reply:
Okay I see it needs one more change. Thanks for pointing out. Fixing this.